pkg/networking/networking_test.go

```diff
@@ -54,8 +55,9 @@ func Test_HttpClient_CallingApiUrl_UsesAuthHeaders(t *testing.T) {
 
 func Test_HttpClient_CallingApiUrl_UsesAuthHeaders_OAuth(t *testing.T) {
 	config := getConfig()
-	userAgent := "James Bond"
 	accessToken := "access me"
+	integrationName := "my-integration"
```

Comment:
why would this test check the user agent? isn't its purpose to check the auth header?

Reply:
It previously checked for the user agent, I changed it to only check for authentication.

These are just leftovers